doc/api/process.markdown

```diff
@@ -218,7 +221,7 @@ Note:
   the terminal mode before exiting with code `128 + signal number`. If one of
   these signals has a listener installed, its default behavior will be removed
   (Node.js will no longer exit).
-- `SIGPIPE` is ignored by default, it can have a listener installed.
+- `SIGPIPE` is ignored by default. It can have a listener installed.
```

Comment:
Using current node (4.0.0), I'm able to trigger `EPIPE` on OS X with:

    node -e 'for (var i = 0; i < 100; i++) console.log(i)' | head

If I install epipebomb and require it first, it doesn't fix anything.

So, I guess it's useless now? But you can still get `EPIPE` errors? Not sure.